generators/client/templates/angular/src/main/webapp/app/shared/login/login.component.ts.ejs

```diff
@@ -48,7 +48,7 @@ export class <%=jhiPrefixCapitalized%>LoginModalComponent implements AfterViewIn
     }
 
     ngAfterViewInit() {
-        this.renderer.invokeElementMethod(this.elementRef.nativeElement.querySelector('#username'), 'focus', []);
+        setTimeout(() => this.renderer.invokeElementMethod(this.elementRef.nativeElement.querySelector('#username'), 'focus', []));
```

Comment:
set timeout needs a second parameter, in this case it can be 0

Reply:
Instead of setTimeout(), can you use Promise.resolve?

Reply:
IMO using promise here might confuse people, eventhough I get the idea